Configure

```diff
@@ -1834,8 +1834,8 @@ Usage: $me [-dehrsEKOSV] [-f config.sh] [-D symbol] [-D symbol=value]
   -r : reuse C symbols value if possible (skips costly nm extraction).
   -s : silent mode, only echoes questions and essential information.
   -D : define symbol to have some value:
-         -D symbol         symbol gets the value 'define'
-         -D symbol=value   symbol gets the value 'value'
+         -D symbol                symbol gets the value 'define'
+         -D symbol='some value'   symbol gets the 'some value'
```

Comment:
Maybe `symbol is set to 'some value'` ? This reads silly

Reply:
> Maybe `symbol is set to 'some value'` ? This reads silly

@Tux, I'm a bit confused.  You appeared to like this wording [two days ago](https://github.com/Perl/perl5/issues/18573#issuecomment-779099615).  The objective here is simply to illustrate how to handle a value that contains a space.

Reply:
Your confusion is my fault. In my initial approval I only looked at the left side of the change.

`symbol gets the 'some value'

reads weird, that is why I would prefer the slight rewording to `symbol is set to 'some value'`
Very sorry for not paying more attention and causing noise